### DISCUSSION TITLE
v1.1.6

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#273] [Announcements/公告]
<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.1.6'>v1.1.6</a>.</em>

**Comment by shaoying189** [comment]
沙发

**Comment by yucotang** [comment]
复活好友能量这个功能，下面的【不复活好友能量名单】，个人建议改成【复活好友能量名单】，或者可以选择其一：一天最多复活6个好友，一般都会选择复活自己小号，没有小号的另说，好友总数多的好几百，如果只有【不复活好友能量名单】的话需要勾选的太多了

**Comment by cxqd888** [comment]
建议蚂蚁新村捐赠增加一个开关

**Comment by xiaolanchong123** [comment]
感谢大佬默默付出，赞

**Comment by shohoku521** [comment]
夜间保活不了

**Comment by emmming** [comment]
这个怎么说呢 状态栏显示空白  可能是状态栏太宽了 建议该小一点 要不然跟水晶155搭配使用芝麻粒就什么都不显示了

**Comment by mapleshadow** [comment]
好用的不行，太好了

**Comment by mozumin** [comment]
步数无法同步至运动。不知是不是版本问题。
支付宝版本号 V10.5.10

**Comment by LiuYunGuiLang** [comment]
早上睡了个懒觉，起来一看，状态栏在，打开日志一看，一片空白。进入森林一看，一个能量没收。
结束支付宝重新打开，开始收……
